core/rest-client/src/http_client.rs

```diff
@@ -419,128 +450,127 @@ mod tests {
 		assert!(!coins_list.is_empty());
 	}
 
-	fn headers_connection_close() -> Headers {
-		let mut headers = Headers::new();
-		add_to_headers(&mut headers, CONNECTION, HeaderValue::from_str("close").unwrap());
-		headers
-	}
-
-	fn deserialize_response_body<'a, T>(encoded_body: &'a [u8]) -> Result<T, Error>
-	where
-		T: Deserialize<'a>,
-	{
-		serde_json::from_slice::<'a, T>(encoded_body).map_err(|err| {
-			Error::DeserializeParseError(err, String::from_utf8_lossy(encoded_body).to_string())
-		})
-	}
-
-	#[test]
```

Comment:
Removed the encryption tests, which were only there to check how the http-req library handles the different certificates we want to block